lms/djangoapps/instructor/views/api.py

```diff
@@ -1246,7 +1246,7 @@ def get_students_features(request, course_id, csv=False):  # pylint: disable=red
         query_features = [
             'id', 'username', 'name', 'email', 'language', 'location',
             'year_of_birth', 'gender', 'level_of_education', 'mailing_address',
-            'goals'
+            'city', 'country', 'goals'
```

Comment:
@regisb would it be possible to reorder this so that we don't move the "goals" position, but are just strictly adding columns to the end?